lib/rules/web/ocm_console/install_page.xyaml

```diff
@@ -118,37 +113,198 @@ check_evaluation_notification_in_install_page:
+check_download_installer_part_in_install_page:
+    action: check_linux_download_installer_button_in_install_page
+    action: check_macos_download_installer_button_in_install_page
+    elements:
+        - selector:
+            xpath: //h3[text()='OpenShift installer']
+check_linux_download_installer_button_in_dev_preview:
+    action: click_download_installer_button
+    elements:
+        - selector:
+            xpath: //a[text()='Download installer']/../../div/select/option[text()='Linux']
+          op: click
+        - selector:
+            xpath: //a[text()='Download installer' and @href='https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest-4.4/openshift-install-linux.tar.gz']
+check_macos_download_installer_button_in_dev_preview:
+    action: click_download_installer_button
+    elements:
+        - selector:
+            xpath: //a[text()='Download installer']/../../div/select/option[text()='MacOS']
+          op: click
+        - selector:
+            xpath: //a[text()='Download installer' and @href='https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest-4.4/openshift-install-mac.tar.gz']
+check_download_installer_part_in_dev_preview:
+    action: check_linux_download_installer_button_in_dev_preview
+    action: check_macos_download_installer_button_in_dev_preview
+    elements:
+        - selector:
+            xpath: //h3[text()='OpenShift installer']
+click_download_cli_button:
+    element:
+        selector:
+            xpath: //a[text()='Download command-line tools']/../../div/select
+        op: click
+check_linux_download_cli_button_in_install_page:
+    action: click_download_cli_button
+    elements:
+        - selector:
+            xpath: //a[text()='Download command-line tools']/../../div/select/option[text()='Linux']
+          op: click
+        - selector:
+            xpath: //a[text()='Download command-line tools' and @href='https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz']
+check_macos_download_cli_button_in_install_page:
+    action: click_download_cli_button
+    elements:
+        - selector:
+            xpath: //a[text()='Download command-line tools']/../../div/select/option[text()='MacOS']
+          op: click
+        - selector:
+            xpath: //a[text()='Download command-line tools' and @href='https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-mac.tar.gz']
+check_windows_download_cli_button_in_install_page:
+    action: click_download_cli_button
+    elements:
+        - selector:
+            xpath: //a[text()='Download command-line tools']/../../div/select/option[text()='Windows']
+          op: click
+        - selector:
+            xpath: //a[text()='Download command-line tools' and @href='https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-windows.zip']
+check_download_cli_part_in_install_page:
```

Comment:
Can this pass the test since there will be once click in `check_linux_download_cli_button_in_install_page` and another click in `check_macos_download_cli_button_in_install_page` which will close the dropdown